snapshots/windows/windows.go

```diff
@@ -125,17 +125,21 @@ func (s *snapshotter) Usage(ctx context.Context, key string) (snapshots.Usage, e
 	if err != nil {
 		return snapshots.Usage{}, err
 	}
-	defer t.Rollback()
+	id, info, usage, err := storage.GetInfo(ctx, key)
+	t.Rollback() // transaction no longer needed at this point.
 
-	_, info, usage, err := storage.GetInfo(ctx, key)
 	if err != nil {
 		return snapshots.Usage{}, err
 	}
 
+	path := s.getSnapshotDir(id)
+
 	if info.Kind == snapshots.KindActive {
-		du := fs.Usage{
-			Size: 0,
+		du, err := fs.DiskUsage(ctx, path)
```

Comment:
I haven't read the windows driver in a while, but I thought it did the same VHD magic, so this file usage is representative of the host instead of the VHD?

Reply:
Here is what I get:
```
$ find /c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots | head -n 20
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/bcd.bak
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/bcd.log.bak
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/bcd.log1.bak
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/bcd.log2.bak
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/blank-base.vhdx
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/blank.vhdx
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/Files
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/Files/License.txt
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/Files/ProgramData
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/Files/ProgramData/Microsoft
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/Files/ProgramData/Microsoft/Diagnosis
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/Files/ProgramData/Microsoft/Diagnosis/AsimovUploader
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/Files/ProgramData/Microsoft/Diagnosis/CustomTraceProfiles
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/Files/ProgramData/Microsoft/Diagnosis/DownloadedScenarios
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/Files/ProgramData/Microsoft/Diagnosis/DownloadedSettings
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/Files/ProgramData/Microsoft/Diagnosis/DownloadedSettings/telemetry.ASM-WindowsDefault.json
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/Files/ProgramData/Microsoft/Diagnosis/DownloadedSettings/utc.app.json
/c/ProgramData/containerd/root/io.containerd.snapshotter.v1.windows/snapshots/101/Files/ProgramData/Microsoft/Diagnosis/DownloadedSettings/utc.privacy.diffbase
```